packages/@jsii/dotnet-runtime/src/Amazon.JSII.Runtime/Services/NodeProcess.cs

```diff
@@ -38,8 +38,7 @@ public NodeProcess(IJsiiRuntimeProvider jsiiRuntimeProvider, ILoggerFactory logg
                     StandardInputEncoding = utf8,
                     RedirectStandardOutput = true,
                     StandardOutputEncoding = utf8,
-                    RedirectStandardError = true,
-                    StandardErrorEncoding = utf8
+                    UseShellExecute = false,
```

Comment:
I think it's a good change but usually has implications on being able to locate executables. Have you tested this on windows?

Reply:
* According to [.NET's own documentation](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.redirectstandardoutput?view=net-5.0#remarks), `UseShellExecute = false` is **required** if you set any `Redirect*` to `true`.
* The *default* value of this property is `false` on `.NET Core` (but `true` on `.NET Framework`). This is only making it explicit